### PR TITLE
feat(api): Add automationcenter endpoint with programResource method

### PR DESCRIPTION
**Pull request by tt1991**

### CODE REVIEW

File: api/index.js [hunk 1]
```diff
@@ -109,6 +111,7 @@ SuiteAPI.createWithCache = function(cacheId, options) {
 module.exports = SuiteAPI;
 
 module.exports.Administrator = AdministratorAPI;
+module.exports.AutomatioCenter = AutomationCenterAPI;
```
Comment (gabor-balla-emarsys) [inline api/index.js]: there is a typo here

File: api/index.spec.js [hunk 2]
```diff
@@ -215,6 +217,12 @@ describe('SuiteApi', function() {
         stub: 'FromAdministratorEndpointStub'
       },
       {
+        endpoint: 'AutomationCenter',
+        sdkKey: 'automationcenter',
+        api: AutomationCenterAPI,
+        stub: 'FromAutomationCenterEndpointStub'
+      },
+      {
```
Comment (gabor-balla-emarsys) [inline api/index.spec.js]: `sdkKey: 'automationcenter'`
shouldn't this be snake cased?

File: README.md [hunk 3]
```diff
@@ -195,6 +195,12 @@ necessary data on the invitation form.
 
     suiteAPI.administrator.inviteExistingAdministrator(payload);
 
+#### AutomationCenter
+
+##### [ProgramResource](http://documentation.emarsys.com/resource/developers/endpoints/automation-center/program-resources/)
+
+    suiteAPI.automationcenter.programResource(payload);
```
Comment (gabor-balla-emarsys) [inline README.md]: `automationcenter`
shouldn't we use snake case here?

**Comment by tt1991** [comment]
I've fixed them.

**Comment by gabor-balla-emarsys** [comment]
I'm really sorry, but I meant camel case instead of snake

File: api/endpoints/automationcenter/index.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+_.extend(AutomationCenter.prototype, {
+
+  programResource: function(payload, options) {
+    return this._requireParameters(payload, ['service_id']).then(function() {
+      logger.log('automationcenter_programresource');
+
+      var url = util.format('/programresource/service_id=%s', payload.service_id);
+      if (payload.resource_id) {
+        url += util.format('&resource_id=%d', payload.resource_id);
+      }
```
Comment (adambrunner) [inline api/endpoints/automationcenter/index.js]: I'd make this here:
```javascript
if (payload.resource_id) {
  var url = util.format('/programresource/service_id=%s&resource_id=%d', payload.service_id, payload.resource_id);
} else {
  var url = util.format('/programresource/service_id=%s', payload.service_id);
}
```

**Comment by tt1991** [comment]
Done

**Comment by adambrunner** [comment]
LGTM

**Pull request merged**